### DISCUSSION TITLE
Cloudflare not connecting properly to cercal

### DISCUSSION THREAD

**Discussion by Valocai** [vercel/community#5287] [Help]
### Summary

Recently moved from 1 domain an other. The other domain worked perfectly fine with virtually no issues. Ever since changing domains vercal seems to find the A record for A @ 76.76.21.21 and then loose it 5 seconds after. www.mydomain.com works fine but mydomain.com by itself either doesent load or takes forever.
![image](https://github.com/vercel/community/assets/84250638/7b1905ec-0033-42eb-8fd8-58126edd1728)
![image](https://github.com/vercel/community/assets/84250638/43ae9ba0-06cb-4331-a7f9-ac562f15902f)
![image](https://github.com/vercel/community/assets/84250638/65f2d54c-646d-4107-8c35-8f43045f6509)
![image](https://github.com/vercel/community/assets/84250638/eaa6f435-bfaa-4409-befa-c4c0bd4f441e)

### Steps to Reproduce

N/A

**Comment by amyegan** [comment]
Hi @Valocai! I recommend following our guide to set up Cloudflare with Vercel: https://vercel.com/docs/integrations/cloudflare

Please make sure you have the appropriate A and CNAME records set for `mydomain.com` and `www.mydomain.com`. You can find additional troubleshooting recommendations at [vercel.com/docs/projects/domains/troubleshooting](https://vercel.com/docs/projects/domains/troubleshooting)
